Platform/NXP/iMX6ULL_EVK_512MB/AcpiTables/Dsdt-Platform.asl

```diff
@@ -13,7 +13,7 @@
 *
 */
 
-OperationRegion(GLBL,SystemMemory,0x10817000,0x10)
+OperationRegion(GLBL,SystemMemory,0x80817000,0x10)

```

Comment:
>0x80817000 [](start = 34, length = 10)

Can you confirm that this is same address passed to 'globalpage' command, and that this address is not overlapping any other address in the memory map?

Reply:
Yes, this address 0x80817000 is correct one, 0x10817000 is not valid address region for 6ULL